src/Sylius/Bundle/ProductBundle/Doctrine/Query/Function/AbstractJsonFunctionNode.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace Sylius\Bundle\ProductBundle\Doctrine\Query\Function;
```

Comment:
Should we sync namespaces with already existing AST functions?

https://github.com/Sylius/Sylius/blob/1.13/src/Sylius/Bundle/CoreBundle/Doctrine/DQL/DateFormat.php

> Doctrine\Query\Function
> Doctrine\Query\AST\Function
> Doctrine\DQL\Function

This one feels most appropriate as it follows `doctrine/orm` naming schema.
> Doctrine\ORM\Query\AST\Function